nevergrad/optimization/optimizerlib.py

```diff
@@ -1631,6 +1660,10 @@ def __init__(
 
 chainCMAPowell = Chaining([CMA, Powell], ["half"]).set_name("chainCMAPowell", register=True)
 chainCMAPowell.no_parallelization = True
+chainMetaModelSQP = Chaining([MetaModel, SQP], ["half"]).set_name("chainMetaModelSQP", register=True)
+chainMetaModelSQP.no_parallelization = True
+chainMetaModelPowell = Chaining([MetaModel, Powell], ["half"]).set_name("chainMetaModelPowell", register=True)
+chainMetaModelPowell.no_parallelization = True
```

Comment:
you should define those on the run in an opt group

Reply:
They are used inside other optimization methods so I think it's difficult. They will be more and more used in the future (see works in progress with significant improvements in particular for big budget). Memetism rules :-)